File: 0500-keyboard-row/0500-keyboard-row.py
class Solution:
    def findWords(self, words: List[str]) -> List[str]:
        fr = "qwertyuiop"
        sr = "asdfghjkl"
        tr = "zxcvbnm"
        ans = []
        for word1 in words:
            fcount = 0
            scount = 0
            tcount = 0
            word = word1.lower()
            for ch in word:
                if ch in fr:
                    fcount += 1
                    
                elif ch in sr:
                    
                    scount += 1
                    
                elif ch in tr:
                   
                    tcount += 1

            if len(word) == fcount or len(word) == scount or len(word) == tcount:
                ans.append(word1)
        return ans

File: 0500-keyboard-row/0500-keyboard-row.java
class Solution {
    
    public String[] findWords(String[] words) {
        String fr = "qwertyuiop";
        String sr = "asdfghjkl";
        String tr = "zxcvbnm";

        List<String> ans = new ArrayList<>();

        for (String word1 : words) {
            int fcount = 0;
            int scount = 0;
            int tcount = 0;

            String word = word1.toLowerCase();

            for (int i = 0; i < word.length(); i++) {
                char ch = word.charAt(i);

                if (fr.indexOf(ch) != -1) {
                    fcount++;
                } else if (sr.indexOf(ch) != -1) {
                    scount++;
                } else if (tr.indexOf(ch) != -1) {
                    tcount++;
                }
            }

            if (word.length() == fcount ||
                word.length() == scount ||
                word.length() == tcount) {
                ans.add(word1);
            }
        }

        return ans.toArray(new String[0]);
    }
}